q2_quality_filter/test/test_filter.py

```diff
@@ -116,6 +116,82 @@ def test_q_score(self):
         self.assertEqual(sorted(obs), sorted(exp_trunc))
         pdt.assert_frame_equal(stats, exp_trunc_stats.loc[stats.index])
 
+    def test_q_score_real(self):
+        ar = Artifact.load(self.get_data_path('real_data.qza'))
+        view = ar.view(SingleLanePerSampleSingleEndFastqDirFmt)
+        obs_result, stats = q_score(view, min_quality=39,
+                                    min_length_fraction=0.24)
+
+        # all truncated reads shown, manually identified. comments denote why
+        # the sequence is not retained.
```

Comment:
I was a little confused by this, since the first read in `exp_result` has a comment but it is retained. Could you change this comment to something like:

```
# All input reads are represented here in their post-quality filtered form. 
# Reads that are commented out were manually 
# identified as being filtered by the q_score method. For the commented reads, the comments denote 
# why the read is not retained.
```